### DISCUSSION TITLE
Specifying noncanonical pairs when using FARFAR2

### DISCUSSION THREAD

**Discussion by rachel-sg** [RosettaCommons/rosetta#179] [General]
Hi,

I am trying to use FARFAR2 to model a sequence with triplexes in secondary structure. One of the types of triplexes is CGG triplex. Bond between C and the first G is the same as in the canonical base pair, so I can specify that in the dot-bracket notation, however, for the bond between two Gs, I think that I need to use the section 'Noncanonical pairs: Specify individual noncanonical pairs using both residues, the interacting edges (W/H/S for Watson-Crick, Hoogsteen, and sugar, or X for any), and finally the interaction orientation (either C/T for cis/trans or P/A for parallel/antiparallel, or X for any).'. However I do not understand how exactly what should I write here. Does anyone have a working example for any type of noncanonical structure?

**Comment by lyskov** [comment]
Changing category, re-opening.

**Comment by everyday847** [comment]
Hi @rachel-sg, it's a little tough to handle triplexes this way. (It's easier when a residue is only making one type of interaction at once.) I believe this may be possible if you use `-bps_moves false`.

Anyway, there is an example in the documentation: `-obligate_pair_explicit 2 11 W W A` (an antiparallel interaction between the W and W edges of the nucleobases 2 and 11). Do you know the edges interacting between the G/G in question? If not, consider using 

```-obligate_pair_explicit 2 11 X X X```

where 2 and 11 are replaced with the nucleotides in question.
